src/components/images_uploading/DisplayImage.js

```diff
@@ -1,68 +0,0 @@
-//Need to re-do this one!

```

Comment:
I renamed the file from 'DisplayImage.js' to 'DisplayImageList.js', so this one no longer existed